### ISSUE TITLE
I think the TODO list should support the ordered list.

### ISSUE THREAD

**Issue by diguage** [asciidoctor/asciidoctor#3211]
Now, the TODO list only supports the unordered list.
But, sometimes we want to know the task number of TODO list. So, the ordered list should be supported.

**Comment by mindentropy** [comment]
@mojavelinux : I am new to Asciidoc so I might not be well read.

In my use case when I write down a task in my notebook I generally have an ordered list of items, and a checkbox at the end of the list item. So to address an item I check if "item number" is done. 

Also I would not want to use an icon for marker if I am writing a quick text document as I need to find a suitable check box icon image now.

**Comment by mindentropy** [comment]
@mojavelinux : 

> AsciiDoc is not a task list system, and was never designed to be. You can check an item, but it's a static HTML page, so when you refresh it, the check will be gone.

I agree 100% with this that it is not an interactive task system. But the underlying text document is used as a task system, a TODO for ideas, logs etc and is very dynamic. Every day I look at the text document and check the TODO ideas to be tried. I comment what works and what does not in the text document itself. It is like dev notes for me. I believe the underlying text document itself should be self explanatory rather than converting to another document to render. I guess that was the original intention i.e. the document is just plain text.

In my case I convert to PDF or HTML only at the very end or only if I need some kind of a report.
